rpmbuild/copr_rpmbuild/rhsm.py

```diff
@@ -0,0 +1,41 @@
+"""
+Red Hat Subscription Management
+
+Activating Red Hat subscription may take a lot of time and historically, the
+subscription service used to be unreliable, so we should activate the
+subscription only when necessary.
+"""
+
+import os
+import shlex
+from copr_rpmbuild.helpers import run_cmd
+
+
+def activate_subscription():
+    """
+    Activate Red Hat Subscription.
+    """
+    pool = "8a85f9a17c71102f017ce611251c770f"
+    user = "copr-team"
+    # TODO Not sure where to store this password so that it is not readable from
+    # the builder (we give SSH access to users, so it cannot be in ENV)
+    password = "TODO"
```

Comment:
@praiskup this may be a big problem. I can't think of any solution how to provide the password so that it is available here but couldn't be simply displayed (or downloaded as a file) by users with SSH access to the builder.